src/api/helpers.jl

```diff
@@ -253,6 +253,20 @@ function h5p_get_file_space_page_size(plist_id)
     return fsp_size[]
 end
 
+function h5p_set_file_space_strategy(plist_id; strategy = nothing, persist = nothing, threshold = nothing)
+    current = h5p_get_file_space_strategy(plist_id)
+    params = (
+        strategy = isnothing(strategy) ? current[:strategy] : strategy,
+        persist = isnothing(persist) ? current[:persist] : persist,
+        threshold = isnothing(threshold) ? current[:threshold] : threshold
+    )
+    return h5p_set_file_space_strategy(plist_id, params)
```

Comment:
I think it would be simpler and just as effective to simply pass them in:
```
h5p_set_file_space_strategy(plist_id, strategy, persist, threshold)
```
As opposed to adding the additional method below, which I don't think is going to be used  elsewhere.

Reply:
That is true, although, if we really wanted to expose this  extended helper method, we would do it outside the `helpers.jl` file in the midlevel layer and rename the function; the methods in this  file are really supposed to be thin  overlays to the low-level api calls and really  intended for internal or low-level calls, so I don't see a huge benefit to the addition.  

Unless there's a direct need, I think it would be  better to keep things simple and only add additional method extensions as we find a direct need.